mobile/build.gradle

```diff
@@ -113,4 +113,7 @@ dependencies {
     compile 'com.google.android.gms:play-services-gcm:8.4.0'
     apt 'org.jraf:prefs-compiler:1.0.1'
     compile 'org.jraf:prefs:1.0.1'
+    testCompile 'junit:junit:4.12'
+    testCompile "org.robolectric:robolectric:3.0"
```

Comment:
We need robolectric because our json parser is inside the android framework.  We therefore requires a real android vm (emulator or device via instrumentation tests) or a mocked one (robolectric) to use the json parser in tests.